src/virtac/create_csv.py

```diff
@@ -256,81 +283,92 @@ def generate_mirrored_pvs(lattice):
-        ("aIn", "basic", "SR-DI-EMIT-01:VEMIT", "SR-DI-EMIT-01:VEMIT_MEAN", emit[1])
+        (
+            "ai",
+            "basic",
+            "SR-DI-EMIT-01:VEMIT",
+            "SR-DI-EMIT-01:VEMIT_MEAN",
+            emit[1],
+            "I/O Intr",
+        )
     )
     data.append(
         (
-            "aIn",
+            "ai",
             "summate",
             "SR-DI-EMIT-01:HEMIT, SR-DI-EMIT-01:VEMIT",
             "SR-DI-EMIT-01:EMITTANCE",
             sum(emit),
+            "I/O Intr",
         )
     )
     # Electron BPMs enabled.
     bpm_enabled_pvs = lattice.get_element_pv_names("BPM", "enabled", pytac.RB)
     data.append(
         (
-            "Waveform",
+            "wfm",
             "collate",
             ", ".join(bpm_enabled_pvs),
             "EBPM-ENABLED:INTERIM",
-            [0] * len(bpm_enabled_pvs),
+            numpy.zeros(len(bpm_enabled_pvs)),
+            "I/O Intr",
         )
     )
     data.append(
         (
-            "Waveform",
+            "wfm",
             "inverse",
             "EBPM-ENABLED:INTERIM",
             "SR-DI-EBPM-01:ENABLED",
-            [0] * len(bpm_enabled_pvs),
+            numpy.zeros(len(bpm_enabled_pvs)),
+            "I/O Intr",
         )
     )
     # BPM x positions for display on diagnostics screen.
     bpm_x_pvs = lattice.get_element_pv_names("BPM", "x", pytac.RB)
     data.append(
         (
-            "Waveform",
+            "wfm",
             "collate",
             ", ".join(bpm_x_pvs),
             "SR-DI-EBPM-01:SA:X",
-            [0] * len(bpm_x_pvs),
+            numpy.zeros(len(bpm_x_pvs)),
+            "I/O Intr",
         )
     )
     # BPM y positions for display on diagnostics screen.
     bpm_y_pvs = lattice.get_element_pv_names("BPM", "y", pytac.RB)
     data.append(
         (
-            "Waveform",
+            "wfm",
             "collate",
             ", ".join(bpm_y_pvs),
             "SR-DI-EBPM-01:SA:Y",
-            [0] * len(bpm_y_pvs),
+            numpy.zeros(len(bpm_y_pvs)),
+            "I/O Intr",
         )
     )
-    # Tune and vertical emittance refresh PVs.
-    data.append(
-        ("aIn", "refresh", "SR-CS-TFB-01:TIMER", "SR23C-DI-TMBF-01:TUNE:TUNE", 0)
-    )
-    data.append(
-        ("aIn", "refresh", "SR-CS-TFB-01:TIMER", "SR23C-DI-TMBF-02:TUNE:TUNE", 0)
-    )
-    data.append(("aIn", "refresh", "SR-CS-VEFB-01:TIMER", "SR-DI-EMIT-01:VEMIT", 0))
     return data
 
 
 def generate_tune_pvs(lattice):
     """Get the PVs associated with the tune feedback system, the structure of
     data is:
-    set pv: The PV to set the offset to.
-    offset: The PV which the set pv reads the offset from.
-    delta: The PV to get the offset from.
+    set_pv: The PV to set the offset to.
+    offset_pv: The PV which the set pv reads the offset from.
+    delta_pv: The PV to get the offset from.
     """
-    data = [("set pv", "offset", "delta")]
+    data: list[tuple] = [("set_pv", "offset_pv", "delta_pv")]
```

Comment:
Can this be type hinted as list[tuple[str, ...]]? If so, probably best as a single type alias covering all such CSV files, i.e. `type CSVData = list[tuple[str, ...]]` or something like that.  I'm not sure there is an easy way to check that each tuple in a given list is the same length.

Splitting them off into separate types for each (e.g. with NamedTuple) seems like it would probably be excessive for only this code here.

Reply:
Agreed with Phil to use str | int, to allow sorting by integer if desired